File: 02_Vergleiche/If.java
import java.util.Scanner;

public class If {
    public static void main(String[] args) {
        int a = 0, b = 1;

        /*
            Hinter IF muss eine Bedingung angegeben werden.
            Ist diese Wahr wird der folgende Block ausgeführt,
            ist sie falsch wird das Programm nach dem IF fortgesetzt.
        */
        if (a < b) {
            System.out.println("Wahr");
        }

        /*
            Gibt es ein ELSE, so kann auch der Falsch-Fall abgefangen
            werden.
            Es wird immer nur ein Block ausgeführt
        */
        if (a > b) {
            System.out.println("Wahr");
        } else {
            System.out.println("Falsch");
        }

        Scanner s = new Scanner(System.in);
        System.out.print("Bitte geben Sie das Passwort ein: ");
        String passwort = s.nextLine();

        if (passwort.equals("1234")) {
            System.out.println("Richtiges Passwort! Willkommen!");
        } else {
            System.out.println("Alaaaaarm!");
        }
    }
}

File: 02_Vergleiche/If.py
a = 0
b = 1
text = ""

# Hinter IF muss eine Bedingung angegeben werden.
# Ist diese Wahr wird der folgende Block ausgeführt,
# ist sie falsch wird das Programm nach dem IF fortgesetzt.

if a < b:
    print("Wahr")

# Gibt es ein ELSE, so kann auch der Falsch-Fall abgefangen
# werden.
# Es wird immer nur ein Block ausgeführt

if a > b:
    print("Wahr")
else:
    print("Falsch")


# Den Bedingungsoperator gibt es in ähnlicher Form

if b == 1:
    a = 1
else:
    a = 0
# ist gleichbedeutent mit:
a = 1 if b == 1 else 0


text = "a ist kleiner" if a < b else "b ist kleiner oder beide sind gleich"
print("Hallo, Welt!" if a == 0 else "Hello, world!")


passwort = input("Bitte geben Sie das Passwort ein: ")
if passwort == "12345":
    print("Richtiges Passwort! Willkommen!")
else:
    print("Alaaaaarm!")

# oder

print("Richtig" if passwort == "12345" else "Alarm!")
